Basic-Scripts/Geocoding and Reverse Geocoding/geocoding_and_reverse_geocoding.py

```diff
@@ -0,0 +1,72 @@
+import requests
+import sys
+
+# If you want your own private_token please refer the README file for this script.
+private_token = "fd00189046e6f6"
```

Comment:
```suggestion
private_token = "<ENTER PRIVATE TOKEN>"
```
No need to expose tokens